src/PhpSpec/NamespaceProvider/NamespaceLocation.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace PhpSpec\NamespaceProvider;
+
+use Composer\Autoload\ClassLoader;
+
+final class NamespaceLocation
```

Comment:
Tell me if you prefer a simple array instead of this POPO

Reply:
Value objects are good :-)